src/components/editor/Tabs.tsx

```diff
@@ -46,25 +62,29 @@ export class Tabs extends React.Component<{
     e.preventDefault();
   }
 
-  onDoubleClick = (e: MouseEvent<any>) => {
-    return this.props.onDoubleClick && this.props.onDoubleClick();
-  }
+  onDoubleClick   = (e: MouseEvent<any>)  => { this.props.onDoubleClick(); };
+  getContainerRef = (ref: HTMLDivElement) => { this.container = ref; };
```

Comment:
Given that this sets the container ref, wouldn't `setContainerRef` be a better name. Throughout the code base we use this naming convention. In fact we usually use `setContainer`, I don't mind `setContainerRef` but I think we should have a PR where we change all such names, not just here.

Reply:
Actually we retrieve reference and assign it to variable. Never mind, I could rename this to ```setContainerRef```. Much important is moving this process outside of **render** because when we create closure object inside render it always create new one and while reconciliation process results in continuous re-rendering.